log/devlog/devlog.go

```diff
@@ -0,0 +1,127 @@
+// Package devlog provides a plain-text logger suitable for reading from the
```

Comment:
I would call this "plaintext" or "text" or "plain" (not "devlog"). That is, the package name and file names.

Reply:
slog and log already both provide plain text loggers. This is a specific customization (that happens to use plain text instead of JSON), so I think the existing name is better.

Reply:
Sure, I'll agree to disagree.